gatsby-config.js

```diff
@@ -146,12 +147,6 @@ let plugins = [
       defaultDataLayer: { website: 'docs' },
     },
   },
-  {
-    resolve: `gatsby-plugin-percy`,
-    options: {
-      // see actual configuration in `percy` key in `package.json`
```

Comment:
Is that configuration thing in pacakge.json still used? Is it the only way now to configure the plugin or would it make sense to add something similar around line 78?

Reply:
I don't think it's going through it anymore, so I deleted this package/plugin.